assets/src/edit-story/app/story/useStoryReducer/reducers/restore.js

```diff
@@ -26,6 +27,7 @@ function restore( state, { pages, current, selection, story } ) {
 		current: newCurrent,
 		selection: newSelection,
 		story: newStory,
+		capabilities: newCapabilities,
```

Comment:
Please remember to add it to [the initial state](https://github.com/ampproject/amp-wp/blob/feature/document-sidebar1/assets/src/edit-story/app/story/useStoryReducer/useStoryReducer.js#L12-L17) of the reducer.

Also, please remember to update all tests (ensure 100% test coverage in [`reducers`](https://github.com/ampproject/amp-wp/tree/feature/document-sidebar1/assets/src/edit-story/app/story/useStoryReducer/reducers) folder).

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/4122/commits/bae3059721739ba3ffa181c0fda173da440ddff3